src/Components/CommentCard.jsx

```diff
@@ -1,13 +1,35 @@
-import React from 'react'
+import React, {useState} from 'react'
 import './CommentCard.jsx'
 import  './CommentCard.css'
+import { voteOnComment} from './api-utils'
+
+function CommentCard({comment, onVote}) {
+    const [votes, setVotes] = useState(comment.votes)
+    const [voteError, setVoteError] = useState(null)
+
+    const voteHandler = async(voteChange) => {
+        setVotes((prevVotes) => prevVotes + voteChange)
+      
+        
+        try{
```

Comment:
This works very well, well done! Only thing to think about (just a suggestion) but as a user I can up/down vote as many times as I like so may be something you'd want to control :)